src/index.test.js

```diff
@@ -74,59 +93,56 @@ describe('ActiveStorageProvider', () => {
   })
 
   describe('if custom headers are provided', () => {
-    const customHeaders = { 'TEST-HEADER': 'testValue' }
+    const baseCustomHeaders = { 'TEST-HEADER': 'testValue' }
+
+    it('merges headers indifferently', async () => {
+      const customHeaders = {...baseCustomHeaders, 'X-CSRF-Token': 'testToken' },
+            expected = expectedCalledWith(customHeaders)
```

Comment:
I appreciate the impulse to DRY, but I'd prefer the meat of the tests to be self-contained so we don't have to jump around to understand what we're expecting. Can you actually inline the expected values?

Reply:
yeah, no problem